src/cc/tests/integtest.h

```diff
@@ -68,6 +68,15 @@ class QFSTestEnvironment : public ::testing::Environment
     virtual bool Stop() { return true; }
 
     /**
+     * mLogFile is the path to the logfile which contains the output of the
+     * service simulated by this instance of QFSTestEnvironment. These logfiles
+     * are available for review by developers when running tests. On continuous
+     * integration platforms, where inspection is harder, these logfiles are
+     * emitted onto stdout.
+     */
+    string mLogFile;
```

Comment:
The chunkservers and metaserver already write their logs out to stderr. In production, we redirect that using daemontools' multilog. Here, we're redirecting the process stderr to the log file. That's done in the `dup2` call inside each environment.